src/renderer/components/NewProjectModal/RunTranscriptionView.tsx

```diff
@@ -61,21 +72,55 @@ const RunTranscriptionView = ({ closeModal, nextView }: Props) => {
     ) {
       return;
     }
+    setMediaFilePath(currentProject.mediaFilePath);
```

Comment:
Since you only ever set this to be `currentProject.mediaFilePath`, you can just replace all references to `mediaFilePath` with `currentProject.mediaFilePath`, or destructure it at the point where you set it if you prefer i.e. replace

```
setMediaFilePath(currentProject.mediaFilePath)
```

with just

```
const { mediaFilePath } = currentProject;
```